index.css

```diff
@@ -0,0 +1,14 @@
+/* Variant 1 */
+.Parent {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
```

Comment:
@mpysarenko You need to use` justify-content: center ` for this task. For 1 element there is no difference between `space-around` and `center`, but if we have 2 and more elements and want them to be in the center - we will have to  use only `center`. You can see difference between properties for justify-content here: https://codepen.io/team/css-tricks/pen/zzJMGJ